src/epub.ts

```diff
@@ -25,13 +25,15 @@ export class EPub {
 
   public async getChapter(id: string): Promise<string> {
     const rawChapter = await this.getRawChapter(id);
-    return ChapterCleaner.cleanChapter(
+    const { manifest, contentPath, imageRoot, linkRoot } = this;
```

Comment:
Why?

Reply:
because I change a few parameters to an object
it is more extendable, and we don't care about the order of single params

here I made the destructuring assignment because it is more convenient, in my opinion

Reply:
Ehh, ok. I guess that's the first time we differ in opinion. I will merge it anyway and change it when the need arises.